service/s3/s3crypto/doc.go

```diff
@@ -1,67 +1,4 @@
 /*
-Package s3crypto provides encryption to S3 using KMS and AES GCM.
```

Comment:
We should probably keep the rest of this package's doc for now, until the client is removed.